DALK.PL_ANALYZER/Scripts/modernizr-site.js

```diff
@@ -12,8 +12,40 @@
     }
 }
 
+function verticalAlignAllElements()
+{
+    setHeightForMvpAndDesc();
+    setMarginInNotPlayedMatchNavigation();
+}
+
 function setHeightForMvpAndDesc() {
     setMaxHeightForAllElementsInClass(".match__mvp");
     setMaxHeightForAllElementsInClass(".match__description");
 }
 
+function setMarginInNotPlayedMatchNavigation() {
+    var allNotPlayedMatch = document.getElementsByClassName("match__notPlayed");
+    Array.from(allNotPlayedMatch).forEach(function (notPlayedMatch) {
+
+        var navigations = notPlayedMatch.getElementsByClassName("match__navigation");
+        if (navigations != undefined) {
+            var navigation = navigations[0];
+            navigation.style.marginTop = (getMVPMaxHeight() + 4) + "px";
+
+        }        
+    });
+}
+
+function getMVPMaxHeight()
+{
+    var maxCallback = (max, cur) => Math.max(max, cur);
+    var allElements = document.querySelectorAll(".match__mvp");
```

Comment:
Nice that you are checkining vanilla javascript. Is that on purpose or you dont know about jQuery?

Reply:
When i know how to write something in vanilla and it isn't too dificult i like to choice this way.